### PR TITLE
Style: Change 'Object' to 'object'

### PR DESCRIPTION
**Pull request by iSazonov**
## PR Summary

Style fix: Object -> object

There is two fix in strings - I think it is Ok.

## PR Checklist

- [x] [PR has a meaningful title](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
    - Use the present tense and imperative mood when describing your changes
- [x] [Summarized changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] [Change is not breaking](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#making-breaking-changes)
- [x] [Make sure all `.h`, `.cpp`, `.cs`, `.ps1` and `.psm1` files have the correct copyright header](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] This PR is ready to merge and is not [Work in Progress](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---work-in-progress).
    - If the PR is work in progress, please add the prefix `WIP:` to the beginning of the title and remove the prefix when the PR is ready.
- **User-facing changes**
    - [x] Not Applicable
    - **OR**
    - [ ] User-facing [Documentation needed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
        - [ ] Issue filed - Issue link:
- **Testing - New and feature**
    - [x] Not Applicable or can only be tested interactively
    - **OR**
    - [ ] [Make sure you've added a new test if existing tests do not effectively test the code changed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#before-submitting)
        - [ ] [Add `[feature]` if the change is significant or affects feature tests](https://github.com/PowerShell/PowerShell/blob/master/docs/testing-guidelines/testing-guidelines.md#requesting-additional-tests-for-a-pr)

### CODE REVIEW

File: src/System.Management.Automation/engine/parser/tokenizer.cs [hunk 1]
```diff
@@ -1551,7 +1551,7 @@ private void ScanToEndOfCommentLine(out bool sawBeginSig, out bool matchedRequir
 
         #endregion Utilities
 
-        #region Object reuse
+        #region object reuse
```
Comment (SteveL-MSFT) [inline src/System.Management.Automation/engine/parser/tokenizer.cs]: Is there a convention here?  Seems like the uppercase is more consistent with usage in the source code and slightly easier to read

**Review by SteveL-MSFT — CHANGES_REQUESTED** [review]

File: src/System.Management.Automation/engine/parser/tokenizer.cs (hunk 1 above)
Comment (iSazonov) [reply src/System.Management.Automation/engine/parser/tokenizer.cs]: No convention exists. Reverted for better readability.

File: src/System.Management.Automation/engine/parser/tokenizer.cs [hunk 2]
```diff
@@ -1578,7 +1578,7 @@ private string GetStringAndRelease(StringBuilder sb)
             return result;
         }
 
-        #endregion Object reuse
+        #endregion object reuse
```
Comment (SteveL-MSFT) [inline src/System.Management.Automation/engine/parser/tokenizer.cs]: This one needs to be reverted as well

**Review by SteveL-MSFT — CHANGES_REQUESTED** [review]

File: src/System.Management.Automation/engine/parser/tokenizer.cs (hunk 2 above)
Comment (iSazonov) [reply src/System.Management.Automation/engine/parser/tokenizer.cs]: Fixed.

**Review by SteveL-MSFT — APPROVED** [review]

**Comment by iSazonov** [comment]
Waiting #8530

**Pull request merged**